src/App/Lobby/LobbyService.ts

```diff
@@ -47,6 +47,22 @@ export class LobbyService {
     this.setCredential(player, credential);
   }
 
+  public static async renameUser(gameCode: string, player: IPlayerInRoom, newName: string): Promise<void> {
```

Comment:
I don't see you using this. Delete it.

Reply:
https://github.com/freeboardgame/FreeBoardGame.org/blob/d46a426cf36401f5c640cb93071b0e387cc7d748/src/App/Lobby/Room.tsx#L144

Reply:
Oh, you meant leaveRoom.  Ok, removed.